kili/services/label_import/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+label import service
+"""
+
+from pathlib import Path
+from typing import List, Optional, Type, cast
+
+from kili.exceptions import NotFound
+from kili.services.label_import.importer import (
+    AbstractLabelImporter,
+    KiliRawLabelImporter,
+    LoggerParams,
+    YoloLabelImporter,
+)
+from kili.services.types import LabelFormat, LogLevel
+
+
+def import_labels_from_files(  # pylint: disable=too-many-arguments
+    kili,
+    labels_file_path: Optional[str],
+    labels_files: Optional[List[str]],
+    meta_file_path: Optional[str],
+    project_id: str,
+    input_format: str,
+    target_job_name: Optional[str],
+    disable_tqdm: bool,
+    log_level: str,
+    model_name: Optional[str],
+    is_prediction: bool,
+) -> None:
+    """
+    Imports labels from a list of files contained in file path.
+    """
+    if kili.count_projects(project_id=project_id) == 0:
+        raise NotFound(f"project ID: {project_id}")
+
+    if is_prediction and model_name is None:
+        raise ValueError("If predictions are uploaded, a model name should be specified")
+
+    label_importer_class: Optional[Type[AbstractLabelImporter]] = None
+    if input_format in ["yolo_v4", "yolo_v5"]:
```

Comment:
is there a way to use LabelFormat ? So if one day we add another yolo model, we just have to change it in one place and it updates both the Literal type and the list on which iterate here